tests/_data/dummy_projects/with-shrinkwrap/npm-shrinkwrap.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "",
+  "name": "with-shrinkwrap",
```

Comment:
out of scope. please put this in another (more fast-moving) pullrequest.
thank you in advance